### DISCUSSION TITLE
Support for cockroachdb

### DISCUSSION THREAD

**Discussion by Reynadi531** [umami-software/umami#1062] [Ideas]
Hi,

Other than the scalibility feauture of cockroachdb. Ive seen how useful their free serverless database hosting on cockroachdb labs. Its on postgres wire too, need some tweaks to support the database. Hopefully umami will support cockroachdb soon.

Best regards,
Rey

**Comment by mikecao** [comment]
What we support depend on the prisma library we use, https://www.prisma.io/. It looks like CockroachDB is only a preview feature at the moment. On top of that you have to rewrite a bunch of queries due to custom date functions of each database. If anyone wants to pursue a PR it'd be welcome.
